test/box-luatest/gh_6539_log_user_space_empty_or_nil_select_test.lua

```diff
@@ -108,20 +112,24 @@ for _, eng in pairs{'memtx', 'vinyl'} do
         for _, call_fmt in pairs(explicit_fullscan_call_fmts) do
             local call = call_fmt:format(space)
             g.server:eval(call)
-            t.assert_not(g.server:grep_log(expected_log_entry, grep_log_bytes),
-                        ('log must not contain a critical entry about ' ..
-                         'an empty or nil `select` call on a %s '..
-                         'user space with `fullscan = true` option '..
-                         'set'):format(eng))
+            t.helpers.retrying({}, function()
+                t.assert_not(g.server:grep_log(expected_log_entry, grep_log_bytes),
```

Comment:
Wrapping `assert_not` in `retrying` doesn't make any sense: if the pattern is found in the log, then it'll also be found on the next check.

Reply:
Thanks, fixed.